src/monaca.js

```diff
@@ -2320,7 +2337,21 @@
 
         var webpack = this.getWebPackModule();
 
-        webpack(this.getWebPackConfigs(projectDir), function(err, stats){
+        var loadingTimeoutId;
+        function dot() {
+          loadingTimeoutId = setTimeout(function() {
+            process.stdout.write('.');
+            dot();
+          }, (Math.random() * 2000));
+        }
+        dot();
+
+        var webpackProdConfig = require(path.join(projectDir, 'webpack.prod.config.js'));
+
+        webpack(webpackProdConfig, function(err, stats){
+          clearTimeout(loadingTimeoutId);
+          process.stdout.write('\n');
+
           if(err) {
             process.chdir(pwd);
```

Comment:
This was something I should point out before, but what's the purpose of changing current directory here, and on line #2339?